adr/0001-community-discussions.adoc

```diff
@@ -0,0 +1,70 @@
+= Setup Quarkus Github Discussions to enable the broader Community
+
+* Status: proposed
+* Date: 2021-06-26
+
+== Context and Problem Statement
+
+Quarkus community is growing and until now we've catered very well for core contributors and initial early consumers. 
+
+We have multiple communication channels: https://github.com/quarkusio/quarkus[issues], https://groups.google.com/g/quarkus-dev?pli=1[quarkus-dev mailing list], https://quarkusio.zulipchat.com[zulip], https://stackoverflow.com/questions/tagged/quarkus[stackoverflow]
+
+Isues are great for bugs/feature work. Mailing list for design conversations between developers, chat for watercooler style discussions and stackoverflow for user questions.
+
+This setup has issues though, some are:
+
+- zulip chat is used for a lot of users questions but none of that is easily searchable/discoverable so it is very synchronous,
+- people reported that they don't feel okey posting on quarkus-dev or zulips as it is seems focused on dev work and not so much about community events, jobs, conferences, etc.
+- its hard to monitor as contributor who wants to help answer/ask questions.
+
+How can we improve this situaton and enable the broader community to more easily ask questions and find answers - without it all be relying on just a few Quarkus core contributors?
+
+== Scenarios (optional)
+
+User wants to locate a answer to a question - zulip chats does not show up in google search; stackoverflow might but might not have an answer.
```

Comment:
That's what we try to avoid and we constantly monitor SO. Not great as it's a manual process, but we do our best. 

SO gets well indexed by google and now, if you asked the right question, Google displays the answer directly.

Reply:
interesting factoid - githubs wikis are blocked in github.com/robots.txt but discussions are not.

About SO - its not going away but its a chore to monitor and many feel its getting more and more unfriendly because of all its requirements.

I think we'll keep having SO and zulip users but I honestly think github discussions is a better fit.

Reply:
found this thread about SEO on GitHub discussions: https://github.com/github/feedback/discussions/3318

summary:
make sure quarkus.io and other pages link to the discussions page.
give it 90 days to pick up properly by google
success!

until then search from GitHub UI is better.